kolibri/plugins/learn/assets/src/views/content-card-carousel/index.vue

```diff
@@ -213,25 +249,41 @@
   // width of card + gutter
   $card-height = 210px
 
+
   .content-carousel
     margin-top: 1em
-    margin-bottom: 1em
     clearfix()
 
     &-details
+      // vertical-align: bottom
+      // text-align: justify
       clearfix()
+      position: relative
+      margin-bottom: 1em
 
       &-header
         float: left
-        text-align: left
-        margin-bottom: 1em
-        h2
-          margin: 0
-      &-view-all
-        float: right
-        color: white
-        background-color: $core-action-normal
+        text-decoration: none
+        color: $core-text-default
 
+      &-view-more
+        position: absolute
+        right: 0
+        bottom: 0
+
+      &-link-icon
+        // puts drops the center angle of the arrow to about the level of text's midline
+        transform: translateY(30%)
+        display: inline
+
+        ../-header &
```

Comment:
I'm having trouble figuring out what this means/does. It's mostly a flaw of stylus' website that it's not searchable, and I haven't really studied it in depth.

Not sure if I'm in the minority on this, but in general I'd like to avoid "fancy" features when possible, in case we want to migrate away from Stylus eventually.